languages/csharp/sensitivity/exceptions/helpers/Exceptions.tmt

```diff
@@ -0,0 +1,25 @@
+~class NoOpException start~
```

Comment:
missed `using System;` in all helpers

Reply:
Could this be inserted on the code-generation side, since it's needed in every helper?

Reply:
In every helper in this file